darts/models/theta.py

```diff
@@ -337,7 +337,7 @@ def select_best_model(ts: TimeSeries, thetas: Optional[List[int]] = None,
         using the fitted values on the training series `ts`.
 
 
-        Uses 'backtesting.backtest_gridsearch' with 'use_fitted_values=True' and 'metric=metrics.mae`.
+        Uses 'ForecastingModel.gridsearch' with 'use_fitted_values=True' and 'metric=metrics.mae`.
```

Comment:
This function `select_best_model` is basically doing a gridsearch, but it's misplaced IMO, and I think we should remove it if we can.
Instead one option could be to have each model implement a method `default_hyperparams` returning some good ranges of hyper parameters to be iterated on when grid-searching. This way afterwards in `ForecastingModel` we could have only one method `auto_select_model` doing the gridsearch on reasonable hyper-params.

Reply:
I agree, some standardization may be good here. But should we leave this for a separate PR?